tests/schema/main.spec.js

```diff
@@ -13,14 +14,22 @@ function seed() {
 }
 
 (only.length ? only : all).forEach(suite => {
-  describe(`${suite.description} (${suite.file.replace(`${process.cwd()}/`, '')})`, () => {
+  const normalizedFilename = path.normalize(suite.file);
+  const relativeFilename = normalizedFilename.replace(`${process.cwd()}${path.sep}`, '');
+
+  describe(`${suite.description} (${relativeFilename})`, () => {
     suite.tests.forEach(test => {
       if (!process.env.CI && test.online) return;
       if (process.argv.includes('--skip')) {
         if (!test.skip) return;
         delete test.skip;
       }
 
+      if (test.skip) {
```

Comment:
IMO I like to see that tests are skipped in the mocha output. Before the tests were being forced to pass instead of showing that they were skipped. This does not affect the `--skip` flag.